scripts/cluster_network.py

```diff
@@ -268,12 +265,10 @@ def clustering_for_n_clusters(n, n_clusters, custom_busmap=False, aggregate_carr
     else:
         raise AttributeError(f"potential_mode should be one of 'simple' or 'conservative' but is '{potential_mode}'")
 
-    if custom_busmap:
-        busmap = pd.read_csv(snakemake.input.custom_busmap, index_col=0, squeeze=True)
-        busmap.index = busmap.index.astype(str)
-        logger.info(f"Imported custom busmap from {snakemake.input.custom_busmap}")
```

Comment:
Another way than moving this elsewhere, would be to pass a file through the `custom_busmap` kwarg.

e.g. `busmap = pd.read_csv(custom_busmap, index_col=0, squeeze=True)`

Reply:
isn't that exactly how it is now? if not, sorry, then I don't get your point.